xtevent/_eventols.ado

```diff
@@ -389,14 +590,17 @@ program define _eventols, rclass
 		cap confirm var __k
 		if !_rc qui drop __k
 		if "`methodt'"=="ols" qui drop _ttrend
+		if "`saveint'"!="" qui drop _interact*

```

Comment:
@Constantino-Carreto-Romero I remember we have an error message somewhere to reserve the _k variable names for the program in case the user has some variables that start with _k already. We should probably add a similar check for _interact as well.